ui/desktop/app/routes/scopes/scope/projects.js

```diff
@@ -76,41 +70,45 @@ export default class ScopesScopeProjectsRoute extends Route {
   }
 
   willDestroy() {
-    this.job?.stop();
-    super.willDestroy();
+    super.willDestroy(...arguments);
+    this.poller.cancelAll();
   }
 
   /**
    * Poll for new sessions with credentials. Sends a notification for each new session that has a credential.
    */
-  async poller() {
+  poller = restartableTask(async () => {
```

Comment:
Shouldn't this task be a droppable one rather than restarted? Theoretically if the request took longer than 2 seconds, it would never finish each time since it would restart.

Reply:
hm i don't think this will be an issue since `perform` is only called from within this task function and also from within the model hook  and in those cases previous instances of the task should be canceled. Thoughts?

Reply:
Ah yeah I think I forgot each poll isn't doing a `perform` and was thinking each request would be a new task, this makes sense